### ISSUE TITLE
attribute-hyphenation is not applied to bind attributes

### ISSUE THREAD

**Issue by yekver** [vuejs/eslint-plugin-vue#931]
**Tell us about your environment**	
* **ESLint version: 6.0.1**	
* **eslint-plugin-vue version: 5.2.3**	
* **Node version: 12.6.0**	

**Please show your full configuration:**

```json
{
  "root": true,
  "extends": [
    "eslint:recommended",
    "plugin:vue/recommended",
    "plugin:@typescript-eslint/recommended",
    "plugin:prettier/recommended",
    "prettier/vue"
  ],
  "plugins": ["prettier", "@typescript-eslint", "vue"],
  "env": {
    "node": true
  },
  "parser": "vue-eslint-parser",
  "parserOptions": {
    "parser": "@typescript-eslint/parser"
  },
  "rules": {
    "prettier/prettier": ["error", {
        "singleQuote": true,
        "trailingComma": "all"
    }],
    "vue/require-component-is": "off",
    "vue/attribute-hyphenation": ["error", "always"],
    "vue/require-prop-types": "error",
    "vue/attributes-order": "error",
    "vue/html-quotes": ["error", "double"],
    "vue/order-in-components": "error",
    "vue/component-name-in-template-casing": ["error", "PascalCase", {
      "registeredComponentsOnly": true
    }],
    "one-var": ["error", "never"],
    "curly": "error",
    "padding-line-between-statements": ["error",
      { "blankLine": "always", "prev": "*", "next": "return" },
      { "blankLine": "always", "prev": ["const", "let", "var"], "next": "*"},
      { "blankLine": "always", "prev": "*", "next": ["const", "let", "var"]},
      { "blankLine": "always", "prev": "*", "next": "if" },
      { "blankLine": "always", "prev": "directive", "next": "*" },
      { "blankLine": "any",    "prev": "directive", "next": "directive" },
      { "blankLine": "any",    "prev": ["const", "let", "var"], "next": ["const", "let", "var"]}
    ],
    "eqeqeq": ["error", "always"],
    "no-console": ["error", { "allow": ["warn", "error"] }],
    "no-dupe-args": 0,
    "no-sequences": "error",
    "no-undef": "error",
    "no-shadow": "error",
    "@typescript-eslint/array-type": "error",
  }
}
```

**What did you do?**	

```vue
<MyComponent
      v-bind:testAttr="true"
      :testAttrOne="true"
      testAttrTwo
      testAttrThree="true"
/>
```

**What did you expect to happen?**
first two strings with arguments should generate errors.

**What actually happened?**
I see only this two errors:
`Attribute 'testAttrTwo' must be hyphenated.eslint(vue/attribute-hyphenation)`
`Attribute 'testAttrThree' must be hyphenated.eslint(vue/attribute-hyphenation)`

**Comment by Tsuyoshi84** [comment]
I found out the cause of this problem. 

In my case, I installed both `@nuxtjs/eslint-config` and `vue-eslint-parser`. So, `package.json` was like this:

```package.json
{
  "devDependencies": {
      "@nuxtjs/eslint-config-typescript": "^0.1.3",
      "vue-eslint-parser": "^6.0.4",
  }
}
```

And `yarn.lock` looked like this:

```
"@nuxtjs/eslint-config@1.1.2":
  ...
  dependencies:
    eslint-plugin-vue "^5.2.3"

eslint-plugin-vue@^5.2.3:
  ...
  dependencies:
    vue-eslint-parser "^5.0.0"


vue-eslint-parser@^5.0.0:
  ...

vue-eslint-parser@^6.0.4:
  ...

```

Despite the fact that `eslint-plugin-vue` was supposed to depend `vue-eslint-parser@^5.0.0`, I installed `vue-eslint-parser@^6.0.4` and it caused the mismatch of the parser.

Since the structure of `VDirectiveKey` is different between `vue-eslint-parser@5` and `vue-eslint-parser@6`, `eslint-plugin-vue` coudn't find the right property that is required to detect the attribute-hyphenation error.

So, I just uninstalled `vue-eslint-parser@^6.0.4`, and it works now.
